app/controllers/users_controller.rb

```diff
@@ -51,6 +51,24 @@ def destroy
     end
   end
 
+  def set_current_deck
+
+    # nil передается если нужно снять текущую колоду
```

Comment:
i think what better will do separate method remove_current_deck, and separate logic between this methods.